src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mlp_seq.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/mlp.hpp"
+
+#include "utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+namespace {
+
+std::vector<std::vector<InputShape>> inputShape_2D(bool with_dynamic = true) {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
+        {{1, 64}},
+        {{2, 64}},
+        {{4, 64}},
+        {{8, 64}});
+    if (with_dynamic) {
+        shapes.push_back({{PartialShape{-1, 64}, {{1, 64}, {8, 64}, {8, 64}, {6, 64}}}});
+        shapes.push_back({{PartialShape{-1, 64}, {{2, 64}, {2, 64}, {4, 64}, {3, 64}}}});
```

Comment:
1. Let's place the same shapes not after each other: in this scenario, [the plugin just skips](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/node.cpp#L228) `prepareParams()` stage, so we don't cover primitive caching by such test cases
2. 2 dynamic shapes instances are so similar, I believe we can easily remove one of them without losing the test coverage
```suggestion
        shapes.push_back({{PartialShape{-1, 64}, {{1, 64}, {8, 64}, {6, 64}, {8, 64}}}});
```

cc @a-sidorova

Reply:
Applied